terraform/cloudwatch.tf.json.template.py

```diff
@@ -16,6 +22,20 @@
 )
 
 
+def alarm_resource_name(threshold: MetricThreshold) -> str:
+    # FIXME: Remove this hack of slicing off the prefix from the handler name
+    #        once the Lambda's have been renamed.
+    #        https://github.com/DataBiosphere/azul/issues/5337
+    handler_name = threshold.handler_name
+    if handler_name is not None and handler_name.startswith(threshold.lambda_name):
+        handler_name = handler_name[len(threshold.lambda_name):]
+    return '_'.join((
+        threshold.lambda_name,
+        *((handler_name,) if handler_name else ()),
```

Comment:
Conflating None and empty string here as well. Please don't think that these unforced errors are somehow acceptable. This is now the ninth time I am requesting changes on this PR. For a 125 line change, this is an embarrassingly high number and and a disservice to the team.

Also please assert that handler_name is not `''`.